cmd/kueuectl/app/stop/stop.go

```diff
@@ -24,8 +24,9 @@ import (
 )
 
 const (
-	stopExample = `  # Stop the workload 
-  kueuectl stop workload my-workload`
+	stopExample = `  
```

Comment:
It makes it slightly more readable in code, but maybe it was formatted like that for better display in help?
cc @mbobrovskyi

Reply:
Yes, exactly.

Reply:
I mimicked from kubectl, e.g.  https://github.com/kubernetes/kubectl/blob/3aa701d1293215b887e9f3f0aa2974b24f8162db/pkg/cmd/debug/debug.go#L80-L106 but I'm fine to revert it.

Reply:
We also can try to use [templates.LongDesc](https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/get/get.go#L89). It should format it better.

Reply:
Yeah, agree. I can change it to use it ^^.

Reply:
Thanks! And also don't forget to build it for site using `make generate-kueuectl-docs`

Reply:
Updated.